content/bayes_nn/bayes.md

```diff
@@ -0,0 +1,318 @@
+# Bayesian Neural Network
+
+The usual Neural Network are optimized in way to get fixed value of weights and biases that allows the model perform specific task successfully. Instead in 
+Bayesian Neural Network the weights and biases are the distribution, this type of model could be treated as a ensemble of many neural networks trained by the Bayesian inference.
+
+Bayesian approach for the neural networks allows to estimate the uncertainty and make the decision of the model more robust according to the input data.
+
+
+
+### Difference between usual NN and BNN
+
+
+![Placeholder](../images/bayes_nn/diff.png)
+
+
+### Training of NN and BNN
+=== "NN"
+    ![Placeholder](../images/bayes_nn/trainingNN.png)
+    The parameters  ![formula](https://render.githubusercontent.com/render/math?math=\theta ) are optimized in order to minimaze the loss function
+
+=== "BNN"
+    ![Placeholder](../images/bayes_nn/bayesNN.png)
+     The process is to learn the probability distributions for weights and biases that maximize the likelihood of getting a high probability for the correct data/label ![formula](https://render.githubusercontent.com/render/math?math=D(x,y) ) pairs.
+    The parameters of the weights distributions - mean and standart deviation are the product of the loss function optimization
```

Comment:
"The parameters of the weight distributions, mean and standard deviation, are the results of the loss function optimization."